core/rfb.js

```diff
@@ -1200,22 +1195,22 @@ export default class RFB extends EventTargetMixin {
         // for one of the axes is large enough.
         if (Math.abs(this._accumulatedWheelDeltaX) >= WHEEL_STEP) {
             if (this._accumulatedWheelDeltaX < 0) {
-                this._handleMouseButton(pos.x, pos.y, true, 1 << 5);
-                this._handleMouseButton(pos.x, pos.y, false, 1 << 5);
+                this._handleMouseButton(pos.x, pos.y, true, this._mouseButtonMask | 1 << 5);
+                this._handleMouseButton(pos.x, pos.y, false, this._mouseButtonMask &  ~(1 << 5));
```

Comment:
Do we really need the unset of the bit? Shouldn't this method be in full control of the wheel button bits?

I feel it might imply that the wheel bits might be set in other places as well.